### ISSUE TITLE
Consider supporting FCS version 3.2

### ISSUE THREAD

**Issue by joseph-sch** [DC-analysis/fcswrite#5]
I have just noticed that FCS standard version 3.2 has been published: https://pubmed.ncbi.nlm.nih.gov/32881398/ or http://flowcyt.sourceforge.net/fcs/fcs32.pdf

It is brand new and non-backward compatible so I don't suggest 'switching' altogether, but we should consider supporting users who will want to transition.

**Comment by paulmueller** [comment]
I agree. I would be happy to review a PR (including tests). A kwarg `fcs_version="3.0"` (default for now) or `fcs_version="3.2"` would probably make sense.

**Comment by lin1996-ux** [comment]
Is there any chance to use the kwarg fcs_version in fcswrite.write_fcs(), if I need to write numpy data to a .fcs file in **FCS2.0** file format?

**Comment by paulmueller** [comment]
@lin1996-ux version 2.0 is not supported. The keyword `fcs_version` does not exist yet. Please create a new issue if you need that feature.
